PYChen.md

```diff
@@ -1,3 +1,13 @@
+Last week (05/04 - 05/08)
+* [camera]
+	- add effects UI as apec and fix burst mode bug.
+	- Stauts: all demo functions are ready, but some UI need to polish.
+* [woodduck]
+	- [fix] Bug 1158088 - [Settings] app storage panel doesn't update when user add/remove storage.
+		- adress comment and land it.
+* [others]
+	- `Spark` lightning talk
+
```

Comment:
Thanks for reminding me, since I focused on camera project last week (Becker last 2 week), I only surveyed BT api.
Sean will take more responsibility on this project and I'll re-join BT project after back my honeymoon.